evcache-core/src/main/java/com/netflix/evcache/operation/EVCacheAsciiOperationFactory.java

```diff
@@ -23,5 +28,9 @@ public ExecCmdOperation execCmd(String cmd, ExecCmdOperation.Callback cb) {
         return new ExecCmdOperationImpl(cmd, cb);
       }
 
+    public MutatorOperation mutate(Mutator m, String key, long by, long def,
```

Comment:
nit: would prefer descriptive names like incrementAmount, defaultValue, expiry, opCallback.

Reply:
I kept the nomenclature similar to other op implementations, such as MutatorOperationImpl.